sormas-app/app/src/main/java/de/symeda/sormas/app/sample/edit/SampleEditActivity.java

```diff
@@ -57,6 +70,34 @@ protected Sample buildRootEntity() {
         throw new UnsupportedOperationException();
     }
 
+
+    @Override
+    public List<PageMenuItem> getPageMenuData() {
+        List<PageMenuItem> menuItems = PageMenuItem.fromEnum(SampleSection.values(), getContext());
+        return menuItems;
+    }
+
+    @Override
+    protected BaseEditFragment buildEditFragment(PageMenuItem menuItem, Sample activityRootData) {
+//        return SampleReadFragment.newInstance(activityRootData);
+        int menuKey =0;
+        if(menuItem !=null)
```

Comment:
Please try to adhere to the following code style (we will add a code style guide at a later point in time):
- Empty space between statements like "if" and "for" and the following (
- Always use { and }, even if the body contains only one line